apps/innovations/_services/innovations.service.ts

```diff
@@ -200,14 +200,32 @@ export class InnovationsService extends BaseService {
       .andWhere('support.status <> :supportClosed', { supportClosed: InnovationSupportStatusEnum.CLOSED })
       .getMany();
 
-    // TODO: This will be needed for the notification
-    // const previousAssignedAccessors = supports.flatMap(support =>
-    //   support.userRoles.map(item => ({
-    //     id: item.user.id,
-    //     role: item.role,
-    //     unitId: support.organisationUnit.id
-    //   }))
-    // );
+    // TODO: Will need to add logic to push ASSESSMENT users to implement notification AI04, decision currently on hold by client.
+    const innovation = await em
+      .createQueryBuilder(InnovationEntity, 'innovation')
+      .select(['innovation.status'])
+      .where('innovation.id = :innovationId', { innovationId: innovationId })
+      .getOne();
+
+    if (!innovation) {
+      throw new NotFoundError(InnovationErrorsEnum.INNOVATION_NOT_FOUND);
+    }
+
+    const affectedUsers: {
```

Comment:
Would change this variable to something more related to the accessors, maybe 'previousAssignedAccessors', like it was on the comments

Reply:
I'm actually using a strategy previously used here (by DC if not mistaken), we add all affected users (NA/QA/A) to the list, and then work them inside the handler, by role, so that's why that more broad name.

Reply:
Saw the comment on line 203. Nevermind! Thank you